.github/workflows/lock.yml

```diff
@@ -4,8 +4,14 @@ on:
   schedule:
     - cron: '50 1 * * *'
 
+permissions:
+  contents: read
+
```

Comment:
I believe we can omit this entirely given the permissions declaration in the `job.lock.permissions` sections.

```suggestion
```